code/modules/bitrunning/server/map_handling.dm

```diff
@@ -147,6 +151,9 @@
 	domain_randomized = FALSE
 	retries_spent = 0
 
+	for(var/obj/machinery/computer/security/telescreen/entertainment/tv as anything in SSmachines.get_machines_by_type_and_subtypes(/obj/machinery/computer/security/telescreen/entertainment))
```

Comment:
this proc is a mouthful, i'm curious if we couldn't just put a new proc on `SSmachines` that would work like

```dm
/// Updates all given televisions that we have registered to our subsystem given the arguments
/datum/subsystem/machines/proc/update_all_tvs(whatever_boolean, camera_net)
 for(var/obj/machinery/computer/security/telescreen/entertainment/tv as anything in get_machines_by_type_and_subtypes(/obj/machinery/computer/security/telescreen/entertainment))
  tv.update_shows(whatever_boolean, camera_net)
```

also name the first arg something nice in this supposed-proc because it's not intuitive for me to read (hence why i wrote `whatever_boolean`). it just seems like a good way to lessen fingerprint across the code to just have an easy proc to invoke since this is apparently a common pattern throughout this PR (as well as other usecases I presume)